treeherder/etl/artifact.py

```diff
@@ -30,6 +30,9 @@ def store_text_log_summary_artifact(job, text_log_summary_artifact):
         ignore_conflicts=True,
     )
 
+    # Bulk create doesn't return .id field, so query to get them.
+    log_errors = TextLogError.objects.filter(job=job)
```

Comment:
note, this could be a perf impact, we could reconsider this specific approach, maybe only do this if we have a need to update, or convert bulk_create -> save() in a loop